README.md

```diff
@@ -16,30 +16,34 @@
 [![Coverage Status](https://coveralls.io/repos/github/spyder-ide/spyder/badge.svg?branch=master)](https://coveralls.io/github/spyder-ide/spyder?branch=master)
 [![codecov](https://codecov.io/gh/spyder-ide/spyder/branch/master/graph/badge.svg)](https://codecov.io/gh/spyder-ide/spyder)
 
-----
+![Screenshot of Spyder's main window](./img_src/screenshot.png)
 
-## Important Announcement: Spyder is unfunded!
+----
 
-Since mid November/2017, [Anaconda, Inc](https://www.anaconda.com/) has
-stopped funding Spyder development, after doing it for the past 18
-months. Because of that, development will focus from now on maintaining
-Spyder 3 at a much slower pace than before.
+## Important Announcement: Spyder needs your support!
 
-If you want to contribute to maintain Spyder, please consider donating at
+Since mid November 2017, [Anaconda, Inc](https://www.anaconda.com/) has
+stopped funding Spyder development, after doing so for the past 18
+months. Therefore, without additional funds, development will shift to
+maintaining Spyder 3 at a much slower pace than before, while slowly working
+toward an eventual Spyder 4 feature release sometime in the future.
 
-https://opencollective.com/spyder
+However, with your contribution of effort and funding, we will be able to
+both continue to maintain Spyder 3 at a faster pace, and fund development
+of the new features you've been requesting for Spyder 4 at a greatly
```

Comment:
maybe mentioning improved code completion and debugging as examples could help get ppl excited for spyder 4

Reply:
Yup; I also suggested we mention such on our OpenCollective, etc.